meta.json

```diff
@@ -6,19 +6,33 @@
   "models": [
     {
       "api": "rdk:component:camera",
-      "model": "viam:viamrtsp:rtsp"
+      "model": "viam:viamrtsp:rtsp",
+      "markdown_link": "README.md#configure-your-viamrtsp-camera",
+      "short_description": "configure a rtsp camera that will autodetect the codec to use."
     },
     {
       "api": "rdk:component:camera",
-      "model": "viam:viamrtsp:rtsp-h264"
+      "model": "viam:viamrtsp:rtsp-h264",
+      "markdown_link": "README.md#configure-your-viamrtsp-camera",
+      "short_description": "configure a rtsp camera that uses the H264 codec."
     },
     {
       "api": "rdk:component:camera",
-      "model": "viam:viamrtsp:rtsp-h265"
+      "model": "viam:viamrtsp:rtsp-h265",
+      "markdown_link": "README.md#configure-your-viamrtsp-camera",
+      "short_description": "configure a rtsp camera that uses the H265 codec."
     },
     {
       "api": "rdk:component:camera",
-      "model": "viam:viamrtsp:rtsp-mjpeg"
+      "model": "viam:viamrtsp:rtsp-mjpeg",
+      "markdown_link": "README.md#configure-your-viamrtsp-camera",
+      "short_description": "configure a rtsp camera that uses the M-JPEG codec."
+    },
+    {
+      "api": "rdk:service:discovery",
+      "model": "viam:viamrtsp:onvif",
+      "markdown_link": "README.md#configure-the-viamrtsponvif-discovery-service",
```

Comment:
saw that this was enabled on app so I added it. feels like readmes should be updated with the assumption that the markdown will be read on the config card, but can save that for a later date. 

tested the feature in staging, currently only works with registry modules. 
<img width="1108" alt="Screenshot 2025-01-23 at 10 24 34 PM" src="https://github.com/user-attachments/assets/6abbcc36-51af-4d11-99eb-f22a22e0620a" />

Reply:
These are meant to link to the anchor that shows each model's specific attributes table I believe.

Reply:
The example modules appear to include the model description, example config, attributes, and notes for the model. Easy enough to change it to be only the example config or only the attributes table though.

Reply:
Nah, that's fine then , leave it as is